Packs/Tidy/Integrations/Tidy/Tidy.py

```diff
@@ -653,7 +664,11 @@ def main() -> None:
     hostname = demisto.getArg("hostname") or demisto.getParam("hostname")
     user = demisto.getArg("user") or demisto.getParam("user")
     password = demisto.getArg("password") or demisto.getParam("password")
-    client = TidyClient(hostname=hostname, user=user, password=password)
+    ssh_key = demisto.getArg("ssh_key") or demisto.getParam("ssh_key")
+    if ssh_key:
+        client = TidyClient(hostname=hostname, user=user, password=password, ssh_key=ssh_key)
```

Comment:
lets not call `TidyClient` in the condition, but build the args and then initialize the client after the condition